centralserver/i18n/management/commands/update_pot.py

```diff
@@ -91,10 +91,23 @@ def inside_kalite():
     ignore_patterns = ["*/python-packages/*", "*/kalite/static/*", "*/js/i18n/*.js",
                        "*/i18n/build/*", "*/media/language_packs/*"]
     with inside_kalite():
+        # First make messages for the application
         ensure_dir("locale")
         call_command("makemessages", locale="en", ignore_patterns=ignore_patterns, no_obsolete=True, domain="django")
         call_command("makemessages", locale="en", ignore_patterns=ignore_patterns, no_obsolete=True, domain="djangojs")
-        return glob.glob(os.path.join(''.join(settings.LOCALE_PATHS), "en", "LC_MESSAGES", "*.po"))
+        pofiles = glob.glob(os.path.join(''.join(settings.LOCALE_PATHS), "en", "LC_MESSAGES", "*.po"))
+        # Then for the docs
+        os.chdir(os.path.join("ka-lite-submodule", "sphinx-docs"))
+        p = Popen(["make", "gettext"])
+        p.wait()
+        p = Popen(["sphinx-intl", "update", "-p", "_build/locale", "-l", "en"])
```

Comment:
The central server uses `python2.7`, so it's preferred to use `subprocess.call` in here.